create_static_html_files.py

```diff
@@ -70,7 +78,7 @@ def create_small_html(df_plugins, build_dir):
         name = row['name'] if row['name'] != 'N/A' else 'unknown'
         normalized_name = row['normalized_name'] if row['normalized_name'] != 'N/A' else 'unknown'
 
-        print(display_name, name, normalized_name)
+        logger.info(f'{display_name=} {name=} {normalized_name=}')
```

Comment:
I actually think we should just remove this logging statement. It's not particularly useful for debugging anything imo, and it just adds a lot of noise. Happy to be overruled if we think there are cases where it would be informative!

Reply:
Happy to remove.